### PR TITLE
Role stickiness

### PR DESCRIPTION
**Pull request by Yudai-8-om**
## Description
Applied role stickiness

### Test Case 1
1. run sim
2. move robots and the ball around
3. check stickiness of roles

---
try changing the return of wall_tactic's cost by multiplying that sticky factor by 10, behavior change more obvious
-Kevin

### CODE REVIEW

**Review by kfu02 — CHANGES_REQUESTED** [review]
Can you take care of the GH Actions complaints? Other than that it looks like switch_cost is working, so good to merge once the style complaints are in

**Review by adithya-gv — APPROVED** [review]
Seconding Kevin's changes but otherwise looks good!

**Review by RyanBard7 — CHANGES_REQUESTED** [review]
Fix or clarify the GH action warnings, ex /tactic/wall_tactic.py#L53, or stp/role/assigment/naive.py#L116, etc.

File: rj_gameplay/rj_gameplay/tactic/wall_tactic.py [hunk 1]
```diff
@@ -62,6 +61,16 @@ def unassigned_cost_fn(
         #TODO: Implement real unassigned cost function
         return role.BIG_STUPID_NUMBER_CONST_FOR_UNASSIGNED_COST_PLS_CHANGE
 
+    # def switch_cost_fn(
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/tactic/wall_tactic.py:64]: you can delete this commented out code

File: rj_gameplay/stp/role/assignment/naive.py [hunk 2]
```diff
@@ -71,28 +74,28 @@ def compute_costs_matrix(
             for robot_idx, robot in enumerate(free_robots):
 
                 # If the robot is not visible then do not consider its cost
+                # Cannot be infinte, will break linear_sum_assignment
                 if not robot.visible:
-                    robot_costs[robot_idx, request_idx] = float('inf')
+                    robot_costs[robot_idx, request_idx] = 1e9
```
Comment (kfu02) [inline rj_gameplay/stp/role/assignment/naive.py:79]: nice fix

File: rj_gameplay/stp/role/assignment/naive.py [hunk 3]
```diff
@@ -109,11 +112,11 @@ def compute_costs_matrix(
 
             # Throw an exception if the returned cost is not finite.
             if not isfinite(unassigned_cost):
-                    raise ValueError(
-                        "Got a non-finite cost ({}) for request {} and unassinged robot".format(
-                            cost, request, robot
-                        )
+                raise ValueError(
+                    "Got a non-finite cost ({}) for request {} and unassinged robot".format(
```
Comment (kfu02) [inline rj_gameplay/stp/role/assignment/naive.py]:
```suggestion
                    "Got a non-finite cost ({}) for request {} and unassigned robot {}".format(
```

will fix the style complain about "unused arguments"

**Review by kfu02 — APPROVED** [review]
couple minor things (one of which fixes style complaint brought up by @rkbard ) but nice work overall!

**Review by RyanBard7 — APPROVED** [review]

**Pull request merged**